doc/admin/installation/docker_smallscale.rst

```diff
@@ -114,13 +119,14 @@ Fill the configuration file ``/etc/pretix/pretix.cfg`` with the following conten
     datadir=/data
 
     [database]
-    ; Replace mysql with postgresql_psycopg2 for PostgreSQL
-    backend=mysql
+    ; Replace postgresql_psycopg2 with mysql for MySQL
+    backend=postgresql_psycopg2
```

Comment:
I know, the comment is misleading, but ``postgresql_psycopg2`` is actually deprecated by now in Django, so we can just use ``postgresql`` here instead.